### PR TITLE
dev-uxmt: Policy Traffic model

### PR DESCRIPTION
**Pull request by pbanascisco**
# Pull Request summary:
Policy Traffic model

# Description of changes:
Policy Traffic model

# Checklist:
- [x] Make sure to run pre-commit before committing changes
- [x] Make sure all checks have passed
- [x] PR description is clear and comprehensive
- [x] Mentioned the issue that this PR solves (if applicable)
- [x] Make sure you test the changes

### CODE REVIEW

**Review by smialy — APPROVED** [review]

**Review by jpkrajewski — APPROVED** [review]

File: catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py [hunk 1]
```diff
@@ -0,0 +1,824 @@
...
+
+
+class ServiceTlocList(BaseModel):
+    model_config = ConfigDict(populate_by_name=True, extra="forbid")
+    tloc_list: Optional[RefIdItem] = Field(default=None, validation_alias="tlocList", serialization_alias="tlocList")
+    type: Optional[Global[ServiceType]] = Field(default=None)
+    vpn: Optional[Global[int]] = Field(default=None)
+
+
+ServiceChainType = Literal[
```
Comment (sbasan) [inline catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py:474]: please reuse definitions from models/common.py (fot other items also)

**Review by sbasan — CHANGES_REQUESTED** [review]

File: catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py [hunk 2]
```diff
@@ -0,0 +1,824 @@
...
+    CflowdAction,
+    NatPoolAction,
+    NatAction,
+    SigAction,
+    FallbackToRoutingAction,
+    SseAction,
+]
+
+
+class Sequences(BaseModel):
```
Comment (sbasan) [inline catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py:798]: please add helper methods so user can add items sequentially using builtin types (no additional imports)
check example:
https://github.com/cisco-open/cisco-catalyst-wan-sdk/blob/dev-uxmt/catalystwan/models/configuration/feature_profile/sdwan/topology/custom_control.py#L202
This greatly simplifies item creation process for user:
https://github.com/cisco-open/cisco-catalyst-wan-sdk/blob/dev-uxmt/catalystwan/integration_tests/feature_profile/sdwan/topology/test_topology.py#L62

**Review by sbasan — CHANGES_REQUESTED** [review]

File: catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py (hunk 1 above)
Comment (pbanascisco) [reply catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py]: Done

File: catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py (hunk 2 above)
Comment (pbanascisco) [reply catalystwan/models/configuration/feature_profile/sdwan/application_priority/traffic_policy.py:606]: Done

**Review by sbasan — APPROVED** [review]

**Pull request merged**